framework/behaviors/AttributeTypecastBehavior.php

```diff
@@ -105,8 +105,6 @@
  * $model->typecastAttributes();
  * ```
  *
- * @property Model|BaseActiveRecord $owner the owner of this behavior.
- *
```

Comment:
It has been added to autocompleat by manual . But `php build php-doc/property` remove it.